org.alloytools.alloy.application/src/main/java/edu/mit/csail/sdg/alloy4whole/ExampleUsingTheCompiler.java

```diff
@@ -39,8 +39,8 @@
      * catch them and display them, and they may contain filename/line/column
      * information.
      */
-    public static void main(String[] args) throws Err {
-
+    public static void main(String[] _args) throws Err {
+        String[] args = { "/home/ostera/repos/github.com/ostera/stagevm/spec/models/process.als" };
```

Comment:
What is this?

Reply:
Sorry, remnants of tests I was making. Cleared 🙆‍♂